tictactoe.js

```diff
@@ -0,0 +1,336 @@
+const readline = require('readline');
+
+const readlineInterface = readline.createInterface({
+    input: process.stdin,
+    output: process.stdout
+});
+
+function ask(questionText) {
+    return new Promise((resolve, reject) => {
+        readlineInterface.question(questionText, resolve);
+    });
+};
+
+let ttTable = ['1', '2', '3', '4', '5', '6', '7', '8', '9'];
+//let whichPlayer = player();
+
+start();
+async function start() {
+
+    console.log('\n'
```

Comment:
This code to print the current board state is repeated several times below. Try putting it in a function and then calling that function instead, both here and below.